crates/ruma-common/src/push/action.rs

```diff
@@ -127,13 +127,13 @@ mod tweak_serde {
         },
     }
 
-    #[derive(Clone, PartialEq, Deserialize, Serialize)]
+    #[derive(Clone, Eq, PartialEq, Deserialize, Serialize)]
```

Comment:
I'm wondering whether we use the `PartialEq` implementation at all. Regardless, running the extra derive macro here has no benefit at all, because this is a crate-internal type.

Note to self: File clippy bug that this lint should only apply to exported types.